shell/packages/sandstorm-db/db.js

```diff
@@ -590,7 +592,7 @@ isSignedUpOrDemo = function () {
   return false;
 }
 
-var calculateReferralBonus = function(accountId, plan) {
+var calculateReferralBonus = function(user, plan) {
```

Comment:
Since you're now passing the actual `user` in here (although I'd rather call it `account`), you could stop passing in the `plan` parameter and pull it out of the user.